### DISCUSSION TITLE
Self Reporting Weather Conditions

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#178] [Ideas]
A feature request for self reporting weather conditions. 

Though this would require a database, this would be something to look into maybe in the future.

**Comment by farfromrefug** [comment]
@jacobgonzales20 what would be the use case of this?

**Reply by ghost** [reply]
more accurate weather

**Reply by farfromrefug** [reply]
@jacobgonzales20 how could reporting make it more accurate ? I dont provide a model. I won't report to Météo-France or OpenMeteo ;)
